mlir/unittests/programs/qc_programs.cpp

```
@@ -271,6 +288,32 @@ void inverseMultipleControlledX(QCProgramBuilder& b) {
+
+void powNegHalfX(QCProgramBuilder& b) {
+  auto q = b.allocQubitRegister(1);
+  b.pow(-0.5, [&] { b.x(q[0]); });
+}
+
+void powThirdX(QCProgramBuilder& b) {
+  auto q = b.allocQubitRegister(1);
+  b.pow(1.0 / 3.0, [&] { b.x(q[0]); });
+}
+
+void powThirdXRef(QCProgramBuilder& b) {
```

Cppcheck (2.20.0)

[style] The function 'powThirdXRef' is never used.

(unusedFunction)